### ISSUE TITLE
Admin General Settings Panel - Ensure every form element has a label

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#240]
WCAG 1.3.1,WCAG 3.3.2: Ensures every form element has a label (.o_user_emails)
Tags: Accessibility, WCAG 1.3.1, WCAG 3.3.2, label

Issue: Ensures every form element has a label (label - https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI)

Target application: Odoo - http://localhost:8069/web#action=142&cids=1&id=&menu_id=95&model=res.config.settings&view_type=form

Element path: .o_user_emails

Snippet: `<input class="o_user_emails o_input mt8" type="text" placeholder="Enter e-mail address">`

How to fix: 
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty

Environment: Chrome version 80.0.3987.163

====

Title: WCAG 1.3.1,WCAG 3.3.2: Ensures every form element has a label (.searchInput)
Tags: Accessibility, WCAG 1.3.1, WCAG 3.3.2, label

Issue: Ensures every form element has a label (label - https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI)

Target application: Odoo - http://localhost:8069/web#action=142&cids=1&id=&menu_id=95&model=res.config.settings&view_type=form

Element path: .searchInput

Snippet: <input type="text" class="searchInput" placeholder="Search...">

How to fix: 
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty

Environment: Chrome version 80.0.3987.163

====

These accessibility issues were found using Accessibility Insights for Web 2.16.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
